rust_dev_preview/cross_service/rest_ses/src/work_item/work_item_archived.rs

```diff
@@ -0,0 +1,129 @@
+//! WorkItemArchived represents several ways to specify the archival status of a WorkItem.
+
+use serde::{de, Deserialize, Serialize};
+use std::convert::TryFrom;
+
+use crate::work_item::WorkItemError;
+
+/// An enum to represent the archival status of a WorkItem.
+/// Because this field has varying representations at different parts of the stack, it needs specialized serde visitors.
+/// The Elwing front end uses "active" and "archive".
+#[derive(Clone, Copy, Debug, Default, PartialEq, Eq)]
+pub enum WorkItemArchived {
+    Active,
+    Archived,
+    #[default]
+    All,
+}
+
+const ACTIVE: &str = "active";
+const ARCHIVED: &str = "archived";
```

Comment:
This is the old spec. Elwing now uses true and false.

Reply:
Yes, which I handle in the From<bool> trait impl. For better or worse, this handles the string constants, the numerical values, and the boolean values.